cfgov/v1/feeds.py

```diff
@@ -63,3 +63,29 @@ def serve(self, request, format='html'):
             return FilterableFeed(self, context)(request)
         else:
             return super(FilterableFeedPageMixin, self).serve(request)
+
+
+def get_page_ancestor_with_feed(page):
+    """Return the closest page ancestor with a feed.
+
+    Includes the specified page in the search. Returns None if no ancestors
+    inherit from FilterableFeedPageMixin.
+    """
```

Comment:
Trivial stylistic point: Don't we prefer a blank line after a docstring? (Also below in `get_rss_feed_for_page`.

Reply:
Not that I know of. The PEP [says](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) only that the closing quotes need to be on their own line.